apps/condo/domains/marketplace/access/MarketItem.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Generated by `createschema marketplace.MarketItem 'name:Text; marketCategory:Relationship:MarketCategory:SET_NULL; sku:Text; description:Text; organization:Relationship:Organization:CASCADE;'`
+ */
+
+const { compact, get, uniq } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find, getById } = require('@open-condo/keystone/schema')
+
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadMarketItems ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === RESIDENT) {
```

Comment:
Looks like we need to check access here using the `PriceScope` model. Because this model defines which market items are reachable for residents.
```suggestion
    if (user.type === RESIDENT) {
    // TODO (DOMA-7503) use PriceScope to check access
```